src/measure_theory/group/fundamental_domain.lean

```diff
@@ -71,6 +71,36 @@ lemma mk' (h_meas : null_measurable_set s μ) (h_exists : ∀ x : α, ∃! g : G
       exact hne ((h_exists x).unique hgx hx)
     end }
 
+@[to_additive measure_theory.is_add_fundamental_domain.mk_of_measure_univ_le]
+lemma mk_of_measure_univ_le [is_finite_measure μ] [countable G]
+  (h_meas : null_measurable_set s μ)
+  (h_ae_disjoint : ∀ g ≠ (1 : G), ae_disjoint μ (g • s) s)
+  (h_qmp : ∀ (g : G), quasi_measure_preserving ((•) g : α → α) μ μ)
+  (h_measure_univ_le : μ (univ : set α) ≤ ∑' (g : G), μ (g • s)) :
+  is_fundamental_domain G s μ :=
+{ null_measurable_set := h_meas,
+  ae_disjoint := h_ae_disjoint,
+  ae_covers :=
+  begin
+    replace ae_disjoint : pairwise (ae_disjoint μ on (λ (g : G), g • s)),
```

Comment:
I recently pushed a proof of a similar implication (with `disjoint` instead of `ae_disjoint`) as a part of `exists_ne_one_smul_eq` below. Should we have a lemma about this implication? E.g., you can prove it for any left or right invariant relation on a group.